econml/validate/__init__.py

```diff
@@ -5,7 +5,8 @@
 
 from .drtester import DRTester
 from .results import BLPEvaluationResults, CalibrationEvaluationResults, UpliftEvaluationResults, EvaluationResults
-
+from .sensitivity_analysis import sensitivity_interval, RV, dml_sensitivity_values, dr_sensitivity_values
 
 __all__ = ['DRTester',
-           'BLPEvaluationResults', 'CalibrationEvaluationResults', 'UpliftEvaluationResults', 'EvaluationResults']
+           'BLPEvaluationResults', 'CalibrationEvaluationResults', 'UpliftEvaluationResults', 'EvaluationResults',
+           'sensitivity_interval', 'RV', 'dml_sensitivity_values', 'dr_sensitivity_values']
```

Comment:
Do we want these all to be part of the public API?

Reply:
removed from public